src/openassetio-core/include/openassetio/managerApi/ManagerInterface.hpp

```diff
@@ -304,6 +304,18 @@ class OPENASSETIO_CORE_EXPORT ManagerInterface {
    */
   virtual void initialize(InfoDictionary managerSettings, const HostSessionPtr& hostSession) = 0;
 
+  /**
+   * Clears any internal caches.
+   *
+   * Only applicable if the implementation makes use of any caching,
+   * otherwise it is a no-op. In caching interfaces, this should cause
```

Comment:
[consider][minor] "Should" is a bit of a weasel word, i'd prefer "May" if this is an optional behaviour, or "Will" if it's guaranteed.

Reply:
Copy-pasted docstring, but reasonable argument, I'll update

Reply:
Updated in ca036121

Reply:
> Copy-pasted docstring, but reasonable argument, I'll update

We should maybe actually discuss this in retro, it keeps coming up.

Reply:
I think "must" was the original intention of the API contract. If a manager has caches, it has to clear them when this method is called to be compliant. The only reason cache might not be cleared is if there aren't any...

Reply:
Updated to "will" in d0e8282f